lefthook.yaml

```
@@ -30,5 +30,9 @@ pre-commit:
         pnpm fix_code_quality
         git add {staged_files}
     8_check_type_errors:
-      run: pnpm check_type_errors
-  piped: true
\ No newline at end of file
+      run: pnpm check_type_errors     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)